os/mm/mm_heap/Make.defs

```diff
@@ -65,6 +65,10 @@ ifeq ($(CONFIG_DEBUG_MM_HEAPINFO),y)
 CSRCS += mm_heapinfo.c
 endif
 
+ifeq ($(CONFIG_APP_BINARY_SEPARATION),y)
+CSRCS += mm_partition_mgr.c
```

Comment:
Is this mandatory to separate APP binaries? If it isn't, let's make a new config and cover this with new one.

Reply:
Yes. It is mandatory for app separation since we need to load the apps in different partitions of ram.